website/blog/2022-08-12-model-timing.md

```diff
@@ -0,0 +1,294 @@
+---
+title: "Rooting out resource hogs with the Model Timing tab"
+description: "Monitoring large, complex projects can be difficult. When you're running 1,000+ models in a day, how do you know which of those consistently take the longest to run? In this article, Bennie Regenold and Barr Yaron show the benefits of the Model Timing Tab in dbt Cloud."
+slug: model-timing-tab
+authors: [bennie_regenold, barr_yaron]
+tags: [analytics craft]
+hide_table_of_contents: false
+
+date: 2022-08-19
+is_featured: true
+---
+
+When running a job that has over 1,700 models, how do you know what a “good” runtime is? If the total process takes 3 hours, is that fantastic or terrible? While there are many possible answers depending on dataset size, complexity of modeling, and historical run times, the crux of the matter is normally “did you hit your SLAs”? However, in the cloud computing world where bills are based on usage, the question is really “did you hit your SLAs and stay within budget”? 
+
+Here at dbt Labs, we used the Model Timing Tab in our internal analytics dbt project to help us identify inefficiencies in our incremental dbt Cloud job that eventually led to major financial savings, and a path forward for periodic improvement checks.
```

Comment:
```suggestion
Here at dbt Labs, we used the Model Timing tab in our internal analytics dbt project to help us identify inefficiencies in our incremental dbt Cloud job that eventually led to major financial savings, and a path forward for periodic improvement checks.
```

This is just a suggestion. It's been `Model Timing Tab` or `Model Timing tab` so we should just stick to one consistent version!